gateway/multiauth_test.go

```diff
@@ -255,3 +258,69 @@ func TestMultiSession_BA_Standard_FAILAuth(t *testing.T) {
 		t.Error("Wrong response code received, expected 403: \n", recorder.Code)
 	}
 }
+
+func TestJWTAuthKeyMultiAuth(t *testing.T) {
+	ts := StartTest()
+	defer ts.Close()
+
+	pID := CreatePolicy()
+
+	spec := BuildAndLoadAPI(func(spec *APISpec) {
+		spec.UseKeylessAccess = false
+
+		spec.AuthConfigs = make(map[string]apidef.AuthConfig)
+
+		spec.UseStandardAuth = true
+		authConfig := spec.AuthConfigs["authToken"]
+		authConfig.AuthHeaderName = "Auth-Token"
+		spec.AuthConfigs["authToken"] = authConfig
+		spec.BaseIdentityProvidedBy = apidef.AuthToken
+
+		spec.EnableJWT = true
+		spec.JWTSigningMethod = RSASign
+		spec.JWTSource = base64.StdEncoding.EncodeToString([]byte(jwtRSAPubKey))
+		jwtConfig := spec.AuthConfigs["jwt"]
+		jwtConfig.AuthHeaderName = "Auth-JWT"
+		spec.AuthConfigs["jwt"] = jwtConfig
+		spec.JWTIdentityBaseField = "user_id"
+		spec.JWTPolicyFieldName = "policy_id"
+		spec.JWTDefaultPolicies = []string{pID}
+
+		spec.Proxy.ListenPath = "/"
+	})[0]
+
+	LoadAPI(spec)
+
+	jwtToken := CreateJWKToken(func(t *jwt.Token) {
+		t.Claims.(jwt.MapClaims)["user_id"] = "user"
+		t.Claims.(jwt.MapClaims)["exp"] = time.Now().Add(time.Hour * 72).Unix()
+	})
+
+	key := CreateSession()
```

Comment:
@buger `// Deprecated: Use Test.CreateSession instead.`

Reply:
Missed this change ;)

But test works, so that should be fine.